### PR TITLE
Feature/permissions support

### PR DESCRIPTION
**Pull request by patmccler**
This was part of my "corporate event" work that is the groundwork for adding a permission to a user

This sets us up to add permissions to a user and have those permissions control the pages a user can see.

No more major organizational work should be needed to add other page automation 
I wanted to get this in while working on corp event stuff so other pages can be worked on in parallel

### CODE REVIEW

File: pages/login_page.rb [hunk 1]
```diff
@@ -13,7 +13,7 @@ class LoginPage < Page
     def initialize url:, driver:, wait:
         super
         puts "navigating to login page if not here"
-        @driver.get @url
+        go_to @url
```
Comment (andrew-k9) [inline pages/login_page.rb]: Possibly could be re-named something like "driver_navigate." Naming it something simple could be a problem if we end up using some other library that has a `go_to` method

File: pages/page.rb [hunk 2]
```diff
@@ -15,6 +15,10 @@ def current_path
         uri.path
     end
 
+    def go_to url
```
Comment (andrew-k9) [inline pages/page.rb]: Possibly could be re-named something like "driver_navigate." Naming it something simple could be a problem if we end up using some other library that has a go_to method

**Review by andrew-k9 — APPROVED** [review]
Other than what I said this is good!

File: pages/page.rb (hunk 2 above)
Comment (patmccler) [reply pages/page.rb]: Changed to navigate_to instead of go_to, hopefully that's more specific to navigating on a web page

**Pull request merged**